www/nginx/src/opnsense/mvc/app/controllers/OPNsense/Nginx/forms/upstream.xml

```diff
@@ -40,7 +40,7 @@
   <field>
     <id>upstream.tls_name_override</id>
     <label>TLS: Servername override</label>
-    <help>Use another hostname for SNI and certificate validation.</help>
+    <help>Use another hostname for SNI and certificate validation as $proxy_ssl_name.</help>
```

Comment:
Do we have to explain this with variables?

Reply:
For me personally, it helps a lot to figure out, what the setting does. Otherwise we have to "guess" what changed in `nginx.conf`.

Reply:
may be something like "Override the server name used to verify the Upstream certificate and to be passed to Upstream through SNI (proxy_ssl_name). By default the SNI name from request or $host variable is used"

Reply:
@fichtner Please provide me your preference.

Reply:
How about "Force a specific hostname for the backend connection instead of passing the hostname from the downstream connection to the upstream connection."